siliconcompiler/tools/vpr/place.py

```diff
@@ -19,24 +21,15 @@ def setup(chip):
     chip.set('tool', tool, 'task', task, 'threads', os.cpu_count(),
              step=step, index=index, clobber=False)
 
-    # TO-DO: PRIOROTIZE the post-routing packing results?
     design = chip.top()
     chip.set('tool', tool, 'task', task, 'output', design + '.net', step=step, index=index)
     chip.add('tool', tool, 'task', task, 'output', design + '.place', step=step, index=index)
-    chip.add('tool', tool, 'task', task, 'output', design + '.route', step=step, index=index)
     chip.add('tool', tool, 'task', task, 'output', 'vpr_stdout.log', step=step, index=index)
```

Comment:
not an output?

Reply:
Correct, if you run the placement step by itself, as is the case here, it does not produce a .route file.